Tweak setup.py to not need py-monotonic for python3

https://github.com/harlowja/fasteners/pull/40

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -26,7 +26,7 @@ with open("README.rst", "r") as readme:
 
 install_requires = [
     'six',
-    'monotonic>=0.1',
+    'monotonic>=0.1;python_version<"3.4"',
 ]
 
 setup(
